spec/errors.md

```diff
@@ -44,7 +44,7 @@ or separately by more than one such method.
 When a message contains more than one error,
 or contains some error which leads to further errors,
 an implementation which does not emit all of the errors
-SHOULD prioritise _Syntax Errors_ and _Data Model Errors_ over others.
+MUST prioritise _Syntax Errors_ and _Data Model Errors_ over others.
```

Comment:
I don't think we should take this today. We can make it MUST in 48.

Reply:
I agree, for discussion for v48